TypeScript build failure related to EventEmitter types

### Description

New builds on Travis are currently failing (since 8 days ago) with the following errors:

```
src/adapter.ts:16:40 - error TS2507: Type 'typeof EventEmitter' is not a constructor function type.
16 export class SlackEventAdapter extends EventEmitter {
                                          ~~~~~~~~~~~~
src/http-handler.ts:115:17 - error TS2339: Property 'emit' does not exist on type 'SlackEventAdapter'.
115         adapter.emit('error', error, respond);
                    ~~~~
src/http-handler.ts:117:17 - error TS2339: Property 'emit' does not exist on type 'SlackEventAdapter'.
117         adapter.emit('error', error);
                    ~~~~
src/http-handler.ts:121:17 - error TS2339: Property 'emit' does not exist on type 'SlackEventAdapter'.
121         adapter.emit('error', error);
                    ~~~~
src/http-handler.ts:202:19 - error TS2339: Property 'emit' does not exist on type 'SlackEventAdapter'.
202           adapter.emit(body.event.type, ...emitArguments);
                      ~~~~
Found 5 errors.
```

There seems to be an issue with the type definitions for `eventemitter3`.

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

--- a/packages/events-api/package.json
+++ b/packages/events-api/package.json
@@ -46,7 +46,7 @@
     "@types/debug": "^4.1.4",
     "@types/express": "^4.17.0",
     "@types/lodash.isstring": "^4.0.6",
-    "@types/node": ">=4.2.0",
+    "@types/node": ">=4.2.0 < 13",
     "@types/yargs": "^13.0.0",
     "debug": "^2.6.1",
     "lodash.isstring": "^4.0.1",

